pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -195,21 +195,26 @@ async def run(self):
         for impetus, advisory in advisories.items():
             if not advisory:
                 continue
-            if impetus == "metadata":
+            elif impetus == "metadata":
                 await self.build_and_attach_bundles(advisory)
-                continue
-            await self.sweep_builds_async(impetus, advisory)
+            elif impetus == "prerelease":
+                await self.build_and_attach_prerelease_bundles(advisory)
+            else:
+                await self.sweep_builds_async(impetus, advisory)
 
-        # bugs should be swept after builds to have validation
-        # for only those bugs to be attached which have corresponding brew builds
-        # attached to the advisory
-        # currently for rpm advisory and cves only
+        # bugs should be attached after builds to validate tracker bugs against builds
         _LOGGER.info("Sweep bugs into the the advisories...")
-        self.sweep_bugs()
+        if assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE):
+            self.sweep_bugs(permissive=True)
+        else:
+            self.sweep_bugs()
 
-        _LOGGER.info("Processing attached Security Trackers")
-        for _, advisory in advisories.items():
-            self.attach_cve_flaws(advisory)
+        if 'prerelease' in advisories:
```

Comment:
Thinking that `assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE)` should skip flaw bug processing. There will be RCs that do not have a prerelease advisory, and I think we want to postpone CVE stuff until we're close to GA. 

Or is there a `software_lifecyckle.phase` that would match with the desired here?

Reply:
Yeah that was one of the things I wasn't sure about from the doc if Luke meant us to run cve stuff intermittently or all at end. It makes sense to run all at end, while still reporting on errors with --permissive. I'll make the change

Reply:
Done